bin/install

```diff
@@ -29,7 +29,7 @@ function main
   # Parse command args
   while set -q argv[1]
     switch "$argv[1]"
-      case --help -h '/?'
+      case --help -h
```

Comment:
these aren't a glob, but seems reasonable to remove all the `-?` and `/?` args anyway?

Reply:
Yup, not a glob. I think it's awkward and not very common. That's my humble opinion, though. And I guess you agree on that?

Reply:
I vote just remove it. It is the Windows way of asking for help, but people running OMF on Windows these days are just using WSL or cygwin anyway.